torch/csrc/distributed/rpc/PythonRpcHandler.cpp

```diff
@@ -0,0 +1,96 @@
+#include <torch/csrc/distributed/rpc/PythonRpcHandler.h>
+
+namespace torch {
+namespace distributed {
+namespace rpc {
+
+PyObject* PythonRpcHandler::module_ = nullptr;
+PyObject* PythonRpcHandler::runUDFFunction_ = nullptr;
+PyObject* PythonRpcHandler::loadResultFunction_ = nullptr;
+bool PythonRpcHandler::pyInitializerCalled_ = false;
+
+void PythonRpcHandler::init() {
+  if (!Py_IsInitialized()) {
+    Py_Initialize();
+    pyInitializerCalled_ = true;
```

Comment:
Under what circumstances will we be sending Python UDFs to remote processes that aren't already running Python? Does it simplify matters if you can assume that Python workers always start up from Python?